src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -757,6 +757,10 @@ FixedwingPositionControl::control_auto(const hrt_abstime &now, const Vector2d &c
 		control_auto_position(now, curr_pos, ground_speed, pos_sp_prev, pos_sp_curr);
 		break;
 
+	case position_setpoint_s::SETPOINT_TYPE_VELOCITY:
```

Comment:
SETPOINT_TYPE_VELOCITY will be purged "soon". https://github.com/PX4/PX4-Autopilot/pull/18026

Reply:
@dagar What would be the alternative to pass velocity setpoints? Should we move away from using position setpoint types all together? (Seems fw pos control is the only place this is being used)

Reply:
I'm only talking about the position_setpoint_triplet coming from navigator. Everything else using the `trajectory_setpoint` is still valid.